tests/test_constrained_delaunay.py

```diff
@@ -0,0 +1,39 @@
+import unittest
```

Comment:
Can you move those tests to `shapely/tests/test_constructive.py`? 

(you will also notice that this file doesn't use `unittest`)

Reply:
Any particular reason not to use `unittest` here? Not a huge deal, but I do like the assert API provided through `unittest`. Will move back to similar-style asserts as in the other tests.

I'll also move `test_deluanay` over to `test_constructive` as well